allennlp_config/citation_classification_bert.jsonnet

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
are we aiming to use the same config file for PICO as well ? if so, then maybe rename the file to a more generic name, like `classification_bert.jsonnet`